src/hooks/useMagicAutofocus.js

```diff
@@ -78,6 +93,10 @@ export default function useMagicAutofocus(
             delay = false;
           }, 200);
         });
+      } else if (!isNativeStackAvailable) {
+        // We need to do this in order to assure that the input gets focused
+        // when using fallback stacks.
+        InteractionManager.runAfterInteractions(fallbackRefocusLastInput);
```

Comment:
moved the `fallbackRefocusLastInput` here as it was always calling `triggerFocus` twice.